soh/soh/Enhancements/randomizer/3drando/shops.cpp

```diff
@@ -338,138 +338,138 @@ void InitTrickNames() {
+     Text{"Power Bracelet", "Bracelet de force (prog.)", "Brasaletes progresivos"},
+     Text{"Magic Bracelet", "Bracelet magique (prog.)", "Manoplas progresivas"}};
   trickNameTable[0x82] = {
-     Text{"Progressive Bomb Capacity", "Capacité de bombes (prog.)", "Mayor capacidad de bombas"},
-     Text{"Progressive Bomb Pack", "Paquet de bombes (prog.)", "Zurrón de bombas progresivo"},
-     Text{"Progressive Bomb Box", "Boîte à bombes (prog.)", "Bolsa de bombas progresiva"}};
+     Text{"Progressive Blast Mask", "Masque d'Explosion (prog.)", "Máscara explosiva progresiva"},
+     Text{"Progressive Powder Kegs", "Baril de Poudre (prog.)", "Barril de polvo progresivo"},
+     Text{"Progressive Remote Bombs", "Bombes à distance (prog.)", "Bombas remotas progresivas"}};
   trickNameTable[0x83] = {
-     Text{"Progressive Arrow Capacity", "Capacité de flèches (prog.)", "Mayor capacidad de flechas"},
-     Text{"Progressive Hero's Bow", "Arc du héros (prog.)", "Arco del héroe progresivo"},
-     Text{"Progressive Arrow Holder", "Arbalète (prog.)", "Ballesta progresiva"}};
+     Text{"Progressive Crossbow", "Arbalète (prog.)", "Ballesta progresiva"},
+     Text{"Progressive Sacred Bow", "Arc sacré (prog)", "Arco Sagrado Progresivo"},
+     Text{"Progressive Lynel Bow", "Arc de Lynel (prog.)", "Arco de centaleón Progresivo"}};
   trickNameTable[0x84] = {
-     Text{"Progressive Seed Capacity", "Capacité de graines (prog.)", "Mayor capacidad de semillas"},
+     Text{"Progressive Catapult", "Catapulte (prog.)", "Catapulta progresiva"},
      Text{"Progressive Scattershot", "Lance-Pierre rafale (prog.)", "Resortera múltiple progresiva"},
-     Text{"Progressive Seed Satchel", "Sac de graines (prog.)", "Bolsa de semillas progresiva"}};
+     Text{"Progressive Seed Launcher", "Lanceur de semences (prog.)", "Lanzador de semillas progresivo"}};
   trickNameTable[0x85] = {
-     Text{"Progressive Rupee Capacity", "Capacité de rubis (prog.)", "Mayor capacidad de rupias"},
-     Text{"Progressive Purse", "Sacoche (prog.)", "Cartera de rupias progresiva"},
-     Text{"Progressive Rupee Bag", "Sac à rubis (prog.)", "Zurrón de rupias progresivo"}};
+     Text{"Progressive Rupoor Capacity", "Capacité de Roupir (prog.)", "Capacidad progresiva Rupobre"},
+     Text{"Progressive Spoils Bag", "Sac à Butin (prog.)", "Bolsa de trofeos progresiva"},
+     Text{"Progressive Ruby Bag", "Capacité du sac Ruby (prog.)", "Bolso Ruby progresivo"}};
   trickNameTable[0x86] = {
-     Text{"Progressive Diving Ability", "Plongée (prog.)", "Buceo progresivo"},
-     Text{"Progressive Pearl", "Perle (prog.)", "Perla progresiva"},
-     Text{"Progressive Scute", "Bulle (prog.)", "Fragmento Zora progresivo"}};
+     Text{"Progressive Flippers", "Palmes de zora (prog.)", "Aletas de zora progresiva"},
+     Text{"Progressive Dragon's Scale", "Écaille du dragon d'eau (prog.)", "Escama dragón acuático progresiva"},
```

Comment:
I like the dragon scale, but I'll put back the Pearl as the ALTTP reference instead of the Oxygen Jauge